hpi-schul-cloud/schulcloud-server
# BC-5736 - Redis for tldraw

# Description

QA note:
Change tested on env :
https://bc-5736-redis-for-tldraw.nbc.dbildungscloud.dev/
No issues found. All acceptance criteria done. 
See movie:https://ticketsystem.dbildungscloud.de/browse/BC-5736



## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-5736
https://github.com/hpi-schul-cloud/dof_app_deploy/pull/701


## Changes


## Datasecurity


## Deployment


## New Repos, NPM pakages or vendor scripts


## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
